cmd/kube-controller-manager/app/core.go

```diff
@@ -356,6 +356,7 @@ func startGarbageCollectorController(ctx ControllerContext) (bool, error) {
 	restMapper.Reset()
 
 	config := ctx.ClientBuilder.ConfigOrDie("generic-garbage-collector")
+	config.ContentConfig = dynamic.ContentConfig()
```

Comment:
this is restoring something removed in an earlier commit, just drop this and the earlier delete